S23/thofstrand/lowerCase.py

```diff
@@ -0,0 +1,76 @@
+import pandas as pd
+
+
+
+def lowerCase(word, lang):
+    word = word.strip()
+    langStart = lang[:2]
+
+    if "en" == langStart:
+        lower_word = word.lower()
+        return lower_word
+    
+    if "tr" == langStart or 'az' == langStart:
+        
+        if "I" in word:
+            word_list = [*word]
+            indices = [i for i, x in enumerate(word_list) if x == "I"]
+            
+            for x in indices:
+                word_list[x] = 'ı'
+
+            lower_word = combineToString(word_list)
+            lower_word = lower_word.lower()
+            return lower_word
+        else:
+            return word
+        
+    if "ga" == langStart:
+        lowCase = word.lower()
+        caps = ['A','E','I','O','U','Á','É','Í','Ó','Ú']
+        if word[0] == 'n' or word[0] == 't':
+            if word[1] in caps and word[2].isalpha(): #must check isalpha as Õ parses as `O`,`~`
+                word_list2 = [*lowCase]
+                word_list2.insert(1,'-')
+                lower_word = combineToString(word_list2)
+                return lower_word
+            else:
+                word = word.lower()
+                return word
+            
+        else:
+                word = word.lower()
+                return word
+        
+    if "el" == langStart:
+        if word[-1] == 'Σ':
+            lower_word = word.lower()
+            lower_word_list = [*lower_word]
+            lower_word_list[-1] = 'ς'
+            lower_word = combineToString(lower_word_list)
+            return lower_word
+    else:
+        return word
+    
+def combineToString(char_list):
```

Comment:
I see great use of inbuilt functions, but this combineToDefined defined functions could be used in a single line to reduce extra code.

Reply:
This might be wise but I also feel as though the main function is already way too large. I think splitting it actually makes for fewer lines of code in the long run.

Reply:
Yes, i agree to this then.